longest-substring-without-repeating-characters.cpp

```diff
@@ -0,0 +1,37 @@
+// Author: Feyza Sarı
+// Question Link:
+// https://leetcode.com/problems/longest-substring-without-repeating-characters/
+
+// Sliding Window Algorithm:
+class Solution {
+public:
+    int lengthOfLongestSubstring(string strng) {
```

Comment:
burası için de yine grammarly den vaya elle kontrol etsen güzel olacak,

bir de strng yapmışsın ama soruda s verilmişti ya o orada olduğu gibi kalsa da olur

Reply:
bi önceki editorialımda Kavraz soruda verilen isimleri de değiştirmemi söylediği için o şekilde yapmıştım yine de düzelteyim mi?

Reply:
bence tam soruda oldugu gibi kalmayabilir